tools/bifrost-make-tof-lookup-table.ipynb

```diff
@@ -19,14 +19,12 @@
     "import sciline\n",
```

Comment:
Looks good. Should we move this to the docs, so that make sure the notebook is not broken after updates?